frontend/e-buddy/src/app/(demo)/bots/view/page.tsx

```
@@ -0,0 +1,233 @@
+        <div className="px-4 py-6 bg-red-50 text-red-700 rounded-lg">
+          {error}
+        </div>
+      ) : (
+        <div className="overflow-hidden bg-white shadow-md rounded-lg">
+          {/* Header */}
+          <div className="hidden sm:grid grid-cols-12 gap-4 p-4 bg-gray-100 text-gray-600 font-medium border-b">
+            <div
+              className="col-span-4 flex items-center cursor-pointer"
+              onClick={() => handleSort("name")}
+            >
+              <span>Name</span>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)